app/models/concerns/fog_extensions/google/server.rb

```diff
@@ -12,12 +12,20 @@ def flavors
       end
 
       def image_id
-        image_name unless disks.nil?
+        image_name unless disks.blank?
       end
 
       def vm_description
         pretty_machine_type
       end
+
+      def stop
```

Comment:
no need for these, 1.29.0 supports them: https://github.com/fog/fog/commit/2114827fb7fd977734e29f3569a21965e1cfe231